content/CORE/UIReference/Storage/Pools/DatasetsScreen.md

```diff
@@ -52,7 +55,7 @@ Use **ADVANCED OPTIONS** to add additional dataset settings such as quota manage
 
 Additional **Other Options** Settings
 
-| Setting | Description |
+| Name | Description |
 |---------|-------------|
 | **Read-only** | Select an option from the dropdown list. **On** prevents the dataset being modified. **Off** allows users accessing the dataset to modify its contents. |
 | **Exec** | Select an option from the dropdown list. **On** allows processes to executd from within this dataset. **Off** prevents processes from executing in the dataset. It is recommended to set to **On**. |
```

Comment:
The line below this one
**Snapshot directory ** should be **Snapshot directory**

Reply:
On the table headings, Claire had it correct with | Settings | Description |

Reply:
Table issue (Snapshot directory) has been fixed.
It was my understanding that the style for CORE tables would be "Name | Description" and SCALE would be "Setting  | Description" to further differentiate between them.

Reply:
Never mind Claire. Pay no attention to the now deleted comment.